lib/src/main/java/growthbook/sdk/java/GrowthBookJsonUtils.java

```diff
@@ -38,12 +41,6 @@ private GrowthBookJsonUtils() {
         // FeatureResult
         gsonBuilder.registerTypeAdapter(FeatureResult.class, FeatureResult.getSerializer());
 
-        // Experiment
-        gsonBuilder.registerTypeAdapter(Experiment.class, Experiment.getSerializer());
-
-        // ExperimentResult
-        gsonBuilder.registerTypeAdapter(ExperimentResult.class, ExperimentResult.getSerializer());
```

Comment:
Went back to default serialization. It results in the `conditionJson` being printed as a property, and means we cannot check the whole object, but it'll be easier to maintain.